README.md

```diff
@@ -470,12 +524,19 @@ if they not use it). Having this in mind, to setup a micro-service one
 could do the following:
```

Comment:
nit: I don't think we should keep this part of the doc as is.
- The directives defined in this lib are not seen by the gateway/router nor by the clients.
  - Since they are used on `<...>_DEFINITION` and `<...>OBJECT`, only the services that use them need to know about their existence.

Reply:
hum, I didn't know that. I tested here and you're right. I'll update the README and the examples/federation.ts

Thank you!

Reply:
docs updated